core/src/main/scala/org/apache/spark/deploy/FaultToleranceTest.scala

```diff
@@ -113,7 +112,7 @@ private object FaultToleranceTest extends App with Logging {
     assertValidClusterState()
 
     killLeader()
-    delay(30 seconds)
+    delay(30.seconds)
```

Comment:
If the consesus is we want to keep the time based postfix operators then we can definitetly narrow this down (unfortunatetly those files will remain whitelisted as a whole and won't trigger any warnings so we won't necessarily notice if people start using other postfix operators inside of them).